lib/io/StreamUtils.js

```diff
@@ -0,0 +1,46 @@
+import stream from 'stream';
```

Comment:
these seem OK, but can we add some quick tests?  (e.g. valid empty stream, valid non-empty stream, stream with error)